packages/main/src/DateRangePicker.ts

```diff
@@ -155,6 +159,14 @@ class DateRangePicker extends DatePicker {
 		return CalendarDate.fromTimestamp(this._endDateTimestamp! * 1000).toLocalJSDate();
 	}
 
+	get startValue(): string {
```

Comment:
why do we need these two getters?

Reply:
They're needed for the DateRangePicker, since the calendar in range selection mode now uses ui5-date-ranges instead of ui5-dates. 

`<ui5-date-range start-value="{{startValue}}" end-value="{{endValue}}"></ui5-date-range>`